Applications/Utils/MeshGeoTools/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+INCLUDE_DIRECTORIES(
+	${CMAKE_SOURCE_DIR}
+	${CMAKE_SOURCE_DIR}/BaseLib
+	${CMAKE_SOURCE_DIR}/FileIO
+	${CMAKE_SOURCE_DIR}/FileIO/Legacy
+	${CMAKE_SOURCE_DIR}/GeoLib
+	${CMAKE_SOURCE_DIR}/MathLib
+	${CMAKE_SOURCE_DIR}/MeshLib
+)
```

Comment:
I think none of the include dirs is needed because full paths are specified in the `.cpp` file.